src/lightning/app/cli/commands/cp.py

```diff
@@ -228,7 +279,7 @@ def _sanitize_path(path: str, pwd: str) -> Tuple[str, bool]:
         if path == ".":
             path = pwd
         else:
-            path = os.path.join(pwd, path[1:])
+            path = os.path.join(pwd, path)
```

Comment:
I remember there were a reason for it. Not sure.

Reply:
I added a test for this function, because I was trying to sanitize `r:default-project` from `/` and was getting `/efault-project`.

If the edge case ever pops up again, let's be sure to add to the test.